org.kde.akregator.json

```diff
@@ -0,0 +1,474 @@
+                    "sha256": "d6ad6c6efe130390f854969c420a91d5a87c6fe89fee14e8ca8b32cca358bb3c"
+                }
+            ]
+        },
+        {
+            "name": "kpimtextedit",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DBUILD_TESTING=OFF",
+                "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/release-service/21.12.1/src/kpimtextedit-21.12.1.tar.xz",
+                    "sha256": "d92ace9e4f38b40c6278e813c92e02c90d9a615e541a3efbd92b358a34235ad2"
+                }
+            ]
+        },
+        {
+            "name": "kidentitymanagement",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DBUILD_TESTING=OFF",
+                "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/release-service/21.12.1/src/kidentitymanagement-21.12.1.tar.xz",
+                    "sha256": "9c37162dd4dfbca096d0b656e0ea9b49ae43c30cf92f1cd82ba8b5300d536ae4"
+                }
+            ]
+        },
+
+        {
+            "name": "kimap",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DBUILD_TESTING=OFF",
+                "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/release-service/21.12.1/src/kimap-21.12.1.tar.xz",
+                    "sha256": "9780d770a158cf3415fe7238892498d2dc81ae19bc314adb3d8098dffa7d7ef2"
+                }
+            ]
+        },
+        {
+            "name": "kldap",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DBUILD_TESTING=OFF",
+                "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/release-service/21.12.1/src/kldap-21.12.1.tar.xz",
+                    "sha256": "7d61b4d4dc31870054e984ec34304fbd2f804f63125c4b2df04003f48aa60c8a"
+                }
+            ],
+            "modules": [
+                {
+                    "name": "openldap",
+                    "config-opts": [
+                        "--disable-static",
+                        "--disable-slapd"
+                    ],
+                    "sources": [
+                        {
+                            "type": "archive",
+                            "url": "http://mirror.eu.oneandone.net/software/openldap/openldap-release/openldap-2.4.44.tgz",
```

Comment:
```suggestion
                            "url": "https://mirror.eu.oneandone.net/software/openldap/openldap-release/openldap-2.4.44.tgz",
```

Reply:
Thanks for bringing to notice. I rechecked and found another http link. I've replaced both with https.